docs/kiali/01-01-kiali.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
As Kiali is the new component, we need to define a structure on the website. I can do this later :)